tests/config_dpb_test.py

```diff
@@ -362,99 +498,178 @@ def test_config_breakout_various_modes(self, sonic_db):
         # Ethernet8: start from 4x25G-->2x50G with -f -l
         def config_dpb_port8_4x25G_2x50G_f_l():
 
+            # Input Data
+            interface = 'Ethernet8'
+            curMode = '4x25G[10G]'
+            newMode = '2x50G'
+
+            print("Mocked Child ports data-> {}".format([get_child_ports_mock(interface, curMode), get_child_ports_mock(interface, newMode)]))
+
+            config.get_child_ports = mock.MagicMock( \
+                side_effect = [get_child_ports_mock(interface, curMode), get_child_ports_mock(interface, newMode)])
+
             result = runner.invoke(config.config.commands["interface"].\
-                commands["breakout"], ['Ethernet8', '2x50G', '-v', '-f',\
+                commands["breakout"], ['{}'.format(interface), '{}'.format(newMode), '-v', '-f',\
                  '-l', '-y'], obj=obj)
+
             print(result.exit_code, result.output)
             assert result.exit_code == 0
             assert 'Breakout process got successfully completed.' in result.output
 
             brk_cfg_table = db.cfgdb.get_table('BREAKOUT_CFG')
-            assert brk_cfg_table["Ethernet8"]["brkout_mode"] == '2x50G'
+            assert brk_cfg_table["Ethernet8"]["brkout_mode"] == '{}'.format(newMode)
             return
 
         # Ethernet8: move from 2x50G-->1x100G without force, list deps
         def config_dpb_port8_2x50G_1x100G():
 
+            # Input Data
+            interface = 'Ethernet8'
+            curMode = '2x50G'
+            newMode = '1x100G[40G]'
+
+            print("Mocked Child ports data-> {}".format([get_child_ports_mock(interface, curMode), get_child_ports_mock(interface, newMode)]))
+
+            config.get_child_ports = mock.MagicMock( \
```

Comment:
Trailing backslash should not be necessary due to open parenthesis

Reply:
#done